src/apps/fuzzers/README.md

```diff
@@ -53,6 +53,12 @@ The public API of H3 is covered in the following fuzzers:
 | directedEdgeToCells | [fuzzerDirectedEdge](./fuzzerDirectedEdge.c)
 | originToDirectedEdges | [fuzzerDirectedEdge](./fuzzerDirectedEdge.c)
 | directedEdgeToBoundary | [fuzzerDirectedEdge](./fuzzerDirectedEdge.c)
+| cellsToEdge | [fuzzerEdge](./fuzzerEdge.c)
+| isValidEdge | [fuzzerEdge](./fuzzerEdge.c)
+| edgeToCells | [fuzzerEdge](./fuzzerEdge.c)
+| cellToEdges | [fuzzerEdge](./fuzzerEdge.c)
+| edgeToBoundary | [fuzzerEdge](./fuzzerEdge.c)
+| directedEdgeToEdge | [fuzzerEdge](./fuzzerEdge.c)
```

Comment:
minor nit (and here just because it provides a nice summary): thoughts on the value of a `edgeToDirectedEdges` function?